app/src/app/components/contents/metadata/MetadataListItem.tsx

```diff
@@ -0,0 +1,12 @@
+interface MetadataItemProps {
+  label: string;
+  value: string;
+}
+export const MetadataListItem = ({ label, value }: MetadataItemProps) => {
+  return (
+    <li>
+      <span className="font-semibold mr-2">{label}:</span>
+      <span className="text-muted-foreground">{value}</span>
```

Comment:
Feels like the muted and non-muted text are kind of the wrong way round here, like it should be emphasising the values rather than the titles.

Reply:
hmmm wait isn't this the same as the parameter values? idm switching this around but should we not change the parameter value format too if we switch this?

Reply:
tried it that way but looked very weird.. and in my opinion makes sense to emphasize title